src/main/java/Dao/CartItemDao.java

```diff
@@ -0,0 +1,76 @@
+package Dao;
+
+import Entities.CartItem;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartItemDao extends AbstractDao<CartItem> {
+
+    @Override
+    public CartItem getById(int id) {
+        String SQL_SELECT = "Select * from cart_items where cart_item_id=?";
+        try (PreparedStatement preparedStatement = this.executePrepareStatement(SQL_SELECT)) {
+            preparedStatement.setInt(1, id);
+            ResultSet rs = preparedStatement.executeQuery();
+            if (rs.next()) {
+                return new CartItem(rs.getLong("cart_item_id"), rs.getLong("cart_id"),
+                        rs.getLong("product_id"),
+                        rs.getLong("quantity"));
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Override
+    public List<CartItem> getAll() {
+        List<CartItem> cartItems = new ArrayList<>();
+        String SQL_SELECT_ALL = "Select * from cart_items";
+        try (PreparedStatement ps=this.executePrepareStatement(SQL_SELECT_ALL)) {
+            ResultSet rs=ps.executeQuery();
+            while(rs.next()){
+                cartItems.add(new CartItem(rs.getLong("cart_item_id"),rs.getLong("cart_id"),
+                        rs.getLong("product_id"),  rs.getLong("quantity")
+                ));
+            }
+            return cartItems;
+        } catch(SQLException e){
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Override
+    public void create(CartItem cartItem) {
+        String SQL_INSERT = "Insert into cart_items (cart_item_id, cart_id, product_id, quantity) values(?,?,?,?)";

```

Comment:
I think instead of "Insert" you could have either used INSERT or insert. SQL keywords should be either in fully upper case / fully lower case

Reply:
A quick google search will reveal that SQL keywords are not case sensitive.

Reply:
I know SQL keywords are not case sensitive, therefore you will be able to query your database perfectly even with "Select" / "Update" keywords. But I think its a common practice to keep the SQL keywords either in full uppercase / lowercase. Just a suggestion.
https://towardsdatascience.com/10-sql-standards-to-make-your-code-more-readable-in-2021-4410dc50b909
https://courses.cs.washington.edu/courses/cse154/18sp/18sp-data/documents/styleguide/sql/naming-conventions-sql.html